spec/lucky_flow_spec.cr

```diff
@@ -9,6 +9,7 @@ describe LuckyFlow do
 
     flow.el("@heading", text: "Home").should be_on_page
     flow.el("@heading").should have_text("Home")
+    flow.should have_current_path("/home")
```

Comment:
Sorry to come in late, but could you make a PR that also tests with a Lucky::Action class to make sure the other two method overloads work?

Reply:
We can't write a test for those because this library doesn't actually depend on lucky. If I called those methods it would fail to compile.